src/common.js

```diff
@@ -86,7 +89,8 @@ exports.error = error;
 //@ ```
 //@
 //@ Turns a regular string into a string-like object similar to what each
-//@ command returns. This has special methods, like `.to()` and `.toEnd()`
+//@ command returns. This has special methods, like `.to()` and `.toEnd()
+// XXX: Is this used by anything other than `wrap()`? Does it need to go on plugin.utils?
```

Comment:
So, I'm undecided where to put this.

First of all, does `wrap` auto convert the return value to a shell string?

My gut feeling is that this should be in it's own file (but not it's own plugin), and available as `$.ShellString`